tool.go

```diff
@@ -18,11 +20,14 @@ import (
 	"path"
 	"path/filepath"
 	"runtime"
+	// 	"sort"
```

Comment:
Remove this.

(I guess it's commented out because of the example code being commented out, right? In any case, it's unclear when examples will work, so this may stay here for a while, and it's not helpful. Better to remove it now and add it when it's needed.)